01_export.ipynb

```diff
@@ -0,0 +1,308 @@
+{
```

Comment:
<p>Let me work on this today please, as I already have most of a notebook done</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/muellerzr/fastexport/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>